main.rb

```diff
@@ -0,0 +1,48 @@
+require_relative './app'
+class EntryPoint
+  puts 'Welcome to School Library App'
+
+  def menu
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+end
```

Comment:
- [ ] As required for the project, kindly ensure that yourschool library console app behaves in the same it is as in this [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)  provided for you. Right now, I noticed that only **3 options** seem to work such as `Create a person`, `Create a book`, and `Exit`.
The issue is that when you have created a `book` or a `person` and you want to list all of them you get a message saying that `No people on our list` or `No existing books`. 
And for other options, anyone is working except the listed above. Try to fix that. Below is the gif example of your app:
![UI](https://user-images.githubusercontent.com/83042742/224116524-e9dd3f82-d565-4fe2-86f0-492183133929.gif)